RFS/src/main/java/org/opensearch/migrations/bulkload/common/LuceneDocumentsReader.java

```diff
@@ -161,17 +189,17 @@ protected DirectoryReader wrapReader(DirectoryReader reader, boolean softDeletes
         return reader;
     }
 
-    protected RfsLuceneDocument getDocument(IndexReader reader, int docId, boolean isLive) {
+    protected RfsLuceneDocument getDocument(IndexReader reader, int luceneSegIndex, int luceneDocId, boolean isLive) {
         Document document;
         try {
-            document = reader.document(docId);
+            document = reader.document(luceneDocId);
         } catch (IOException e) {
             log.atError().setCause(e).setMessage("Failed to read document at Lucene index location {}")
-                .addArgument(docId).log();
+                .addArgument(luceneDocId).log();
             return null;
         }
 
-        String id = null;
+        String osDocId = null;
```

Comment:
Oh - I think that I just understood what this is - this is the id for the cluster, not for lucene.  I had thought that 'os' was a 'target' id, like what the value of the docId was after we put it to the target.
Can you make these names a bit longer/clearer?

Reply:
Done